corehq/apps/export/models/new.py

```diff
@@ -1972,10 +1972,6 @@ def schema_version(cls):
     def _get_inferred_schema(cls, domain, app_id, case_type):
         return get_case_inferred_schema(domain, case_type)
 
-    @classmethod
-    def _get_current_app_ids_for_domain(cls, domain, app_id):
-        return get_app_ids_in_domain(domain)
```

Comment:
Why is this removed?  I assume the idea is that it doesn't need all app ids, since related apps are pulled in later anyways?  If so, could you remove the method altogether and replace it with `[app_id] if app_id else []` or something?

Reply:
thats what i did. just removed the subclass implementation, and moved the now shared implementation to the superclass. Although based on Simon's comment I may need to only do that in certain cases.

Reply:
I meant that since `_get_current_app_ids_for_domain` is only referenced in one place and the definition is trivial, you could delete it and in-line the contents.  Was mostly proposing that because the name sounds like it looks up related applications, but all it really does now is stick `app_id` in a list.

Reply:
ah good point, fixed

Reply:
thanks!